libs/init/migrate/sbt/exportscript/src/mill/init/migrate/sbt/ExportSbtBuildScript.scala

```diff
@@ -0,0 +1,225 @@
+package mill.init.migrate.sbt
```

Comment:
How does this relate to the existing `libs/init/sbt/exportplugin/src/mill/main/sbt/ExportBuildPlugin.scala`? Can we consolidate them?